File: src/sdk-reference/php/3/collection/scroll-specifications/snippets/scroll-specifications-1.php

<?php

use \Kuzzle\Kuzzle;

$kuzzle = new Kuzzle('localhost');
$dataCollection = $kuzzle->collection('collection', 'index');

try {
  $res = $dataCollection->scrollSpecifications($scrollId, ['scroll' => '1m']);

  foreach ($res['hits'] as $specification) {
    // Specification
  }
  
  // Total specifications count
  $res['total'];
}
catch (ErrorException $e) {

}

File: src/sdk/android/3/core-classes/collection/scroll-specifications/snippets/scroll-specifications-1.java

Options opts = new Options();
opts.setScroll("1m");

kuzzle
  .collection("collection", "index")
  .scrollSpecifications(scrollId, opts, new ResponseListener<JSONObject>() {
    @Override
    public void onSuccess(JSONObject res) {
      for (int i = 0; i < res.getJSONArray("hits").length(); i++) {
          res.getJSONArray("hits").getJSONObject(i) // Specification
      }

      res.getString("total"); // Total specifications count
    }

    @Override
    public void onError(JSONObject error) {
      // Handle error
    }
  });
